src/components/uac/examples/uac-error/index.njk

```diff
@@ -0,0 +1,64 @@
+---
+layout: none
+---
+{% extends "styles/page-template/_template.njk" %}
+{% from "components/uac/_macro.njk" import onsUACInput %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{% set pageConfig = {
+  "title": "Access Code example"
+} %}
+
+{% block main %}
+    <h1 class="u-mt-l">Start survey</h1>
+
+    {% call
+        onsPanel({
+            title: "There is a problem with your answer",
+            type: "error"
+        })
+    %}
+    <p>These <strong>must be corrected</strong> to continue.</p>
+    {% from "components/lists/_macro.njk" import onsList %}
+    {{
+        onsList({
+            "classes": "list--bare",
+            "itemsList": [
+                {
+                    "text": "Enter an access code to continue",
+                    "index": true,
+                    "url": "#uac",
+                    "classes": "js-inpagelink"
+                }
+            ]
+        })
+    }}
+    {% endcall %}
+
+    {{ onsUACInput({
+        "id": "uac",
+        "name": "uac",
+        "postTextboxLinkText": "Where to find your access code",
+        "postTextboxLinkUrl": "#",
+        "label": {
+            "text": "Enter your 16-character access code",
+            "description": "Keep this code safe. You will need to enter it every time you access your census"
+        },
+        "securityMessage": "Your personal information is protected by law and will be kept confidential.",
+        "error": {
+            "id": "uac",
+            "text": "Enter an access code to continue"
+        }
+    }) }}
+
+    {{ onsButton({
+        "text": "Access survey",
+        "classes": "u-mb-xl"
+    }) }}
+
+    <h2 class="u-fs-r--b u-mb-no">Don't have a code?</h2>
```

Comment:
Can you update this block to match end to end proto but change content to for a generic 'survey' example?...

![Screenshot 2020-06-16 at 15 03 02](https://user-images.githubusercontent.com/43346934/84784778-9c6bc680-afe2-11ea-9a72-540e25178d72.png)